pommerman/graphics.py

```diff
@@ -372,9 +372,19 @@ def render_dead_alive(self):
         dead.width = image_size
         dead.height = image_size
         sprites = []
-
-        for agent in self._agents:
-            x = self.board_right(x_offset=-3) - (4 - agent.agent_id) * (
+        
+        if self._game_type is constants.GameType.FFA:
+            agents = self._agents
+        else:
+            agents = [
```

Comment:
Looks good as is - but as you asked for a suggestion:
```python
agents = [self._agents[i] for i in [0,2,1,3]]
```

Reply:
👍